msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md

```diff
@@ -15,6 +15,8 @@ keywords: teams import messages api graph microsoft migrate migration post
 
 With Microsoft Graph, you can migrate users' existing message history and data from an external system into a Teams channel. By enabling the recreation of a third-party platform messaging hierarchy inside Teams, users can continue their communications in a seamless manner and proceed without interruption.
 
+> **Note:** In the future, Microsoft may require you or your customers to pay additional fees based on the amount of data imported.
```

Comment:
Minor comment, please update the Note tag. Reference: https://docs.microsoft.com/en-us/contribute/markdown-reference

```suggestion
> [!NOTE]
> In the future, Microsoft may require you or your customers to pay additional fees based on the amount of data imported.
```